ansible/roles/ansible-horizon-plugin/tasks/collect_compress.yml

```diff
@@ -36,11 +36,17 @@
   file: path=/tmp/sync_static.py state=absent
 
 - name: Get os_local_settings_path
-  shell: "find {{ horizon_virtual_env }} -type d -path /sys -prune -o -name '*local_settings.d' 2>/dev/null | grep -E 'openstack_dashboard|horizon'"
+  find:
+    paths: /openstack/venvs/
+    file_type: directory
+    use_regex: yes
+    patterns:
+      - '^.*local_settings.d$'
+    recurse: yes
   register: os_local_settings_path
```

Comment:
Find exact local_settings.d. Use single virtual env as 'paths'. paths: /openstack/venvs/ this may contain multiple virtual environments. It may lead to multiple local_settings.d directory.

Reply:
@shyam-biradar we need to check if it returns multiple paths, in that case we need to fail the deployment and ask the user to set the right env path